### PR TITLE
Add IP-Messaging/Rest API

### PR DESCRIPTION
**Pull request by EliuX**
Added all snippets related to the Api Rest code referenced in the CMS

### CODE REVIEW

**Comment by EliuX** [comment]
Check it @mcelicalderon  and make comments on issues.

File: ip-messaging/rest/channels/update-channels/update-channels.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+account = "ACCOUNT_SID"
+token = "AUTH_TOKEN"
+client = TwilioIpMessagingClient(account, token) 
+ 
+# Update the channel
+service = client.services.get(sid="CHANNEL_SID")
+channel = service.channels.create()
+response = channel.update(sid="NEW_SID")
+print response
+
```
Comment (mcelicalderon) [inline ip-messaging/rest/channels/update-channels/update-channels.py]: @EliuX There are three unnecessary spaces here

File: ip-messaging/rest/credentials/delete-credentials/output/delete-credentials.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+A successful request returns an HTTP status code 204 and an empty body.
+
```
Comment (mcelicalderon) [inline ip-messaging/rest/credentials/delete-credentials/output/delete-credentials.json]: empty line

File: ip-messaging/rest/roles/delete-role/output/delete-role.json [hunk 3]
```diff
@@ -0,0 +1,2 @@
+A successful delete will return a status code 204 and an empty body.
+
```
Comment (mcelicalderon) [inline ip-messaging/rest/roles/delete-role/output/delete-role.json]: Empty line

File: ip-messaging/rest/users/delete-user/output/delete-user.json [hunk 4]
```diff
@@ -0,0 +1,2 @@
+A successful delete will return a status code 204 and an empty body.
+
```
Comment (mcelicalderon) [inline ip-messaging/rest/users/delete-user/output/delete-user.json]: Empty line

**Comment by mcelicalderon** [comment]
@EliuX Other than my previous comments only the meta.json files are missing. LGTM other than that

**Comment by EliuX** [comment]
Added `meta.json` for describing snippets to the CMS, and where included some fixes.
Check it @mcelicalderon.

**Comment by jarodreyes** [comment]
@EliuX We're gonna wait to merge this until @atbaker makes some changes to the snippet process. Basically he's gonna make it so that on commits these samples are automatically added to Wagtail in prod.

**Pull request merged**